ament_cmake_uncrustify/cmake/ament_uncrustify.cmake

```diff
@@ -11,12 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 #
 # Add a test to check the code for compliance with uncrustify.
 #
+# The default configuration file used for uncrustify is located at
+# configuration/ament_code_style.cfg within the uncrustify directory
```

Comment:
The file is actually within the `ament_uncrustify` directory.

Nitpick: add a trailing period.

Reply:
Corrected.